mayastor/src/bdev/util/uri.rs

```diff
@@ -54,3 +60,32 @@ pub(crate) fn boolean(
     }
     value.parse::<bool>()
 }
+
+pub(crate) fn uuid(
```

Comment:
This does not seem to be needed ... we already have a Uri::parse() and from there we can just look up key/values by

Reply:
From there where, the fn parse?
From what I can see every device "fetches" its own parameters from the url. So this was just added as a helper that also validates that the UUID is actually valid to avoid duplicating code on every device type.

Reply:
Yes but it does so only once.

Each bdev implements the TryFrom trait. This parses the URL, having functions that accept the URL, parse it, and return it seems, a bit inefficient in that respect, that's mostly the reason for the comment.